docs/api-markdown-documenter/api-documentation-layout.js

```diff
@@ -84,7 +84,7 @@ function layoutContent(apiItem, itemSpecificContent, config) {
 	}
 
 	// Render examples (if any)
-	const renderedExamples = LayoutUtilities.createExamplesSection(apiItem, config);
+	const renderedExamples = LayoutUtilities.createExamplesSection(apiItem, config, "Usage");
```

Comment:
nit: maybe let's move out the string constants `"Usage"` and `"Error Handling"` out into consts at the top of this function so they are easier to find in the future?